### ISSUE TITLE
leave a comment anonymously

### ISSUE THREAD

**Issue by magsout** [webcompat/webcompat.com#1015]
I don't know if it's possible/relevant but when I read this comment https://github.com/webcompat/web-bugs/issues/2480#issuecomment-213571619 maybe it would interesting to talk about it ?

**Comment by miketaylr** [comment]
I think we'll end up with spammers, probably. :/

**Comment by karlcow** [comment]
To give context to @magsout suggestion from the bug is pointing out:

> While I like the idea of an open issue tracker here it’s probably not the most effective way of alerting our team to bugs. Most readers don’t have github accounts so our workflows for bug tracking are tuned to reach people and their complaints in a way that is most accessible, so on Twitter and by email.

Note that these interactions are **not** anonymous: email and twitter. So the issue is more how can webcompat helps users with a different identity handle than github to leave comments on webcompat.com. I agree with Mike, that anonymous == spam. It's already tough for issues opening, I don't want to imagine when it will be about comments.

Maybe let's say we block **anonymous issue reporting** (aka completely the opposite of what @magsout suggests) but find another way to get an identity handle.

Right now the process is 

    anonymous ⟶ webcompat-bot ⇉ github ⇶ closing the bug for lack of details.

Let's say, the person has the choice to authenticate using Twitter to report bugs. At least we have now a better or more reliable system and we could still follow-up with the person.  We can even imagine that we have a twitter bot sending back a notification (if the user decided so) to the user for followup comments.

**Comment by miketaylr** [comment]
> While I like the idea of an open issue tracker here it’s probably not the most effective way of alerting our team to bugs. 

The way I interpret this comment is that anonymous Webcompat bugs are not effective to let developers at The Verge know about bugs. I agree with that -- it took @hallvors doing the outreach.

> Most readers don’t have github accounts so our workflows for bug tracking are tuned to reach people and their complaints in a way that is most accessible, so on Twitter and by email.

Right, most Verge readers don't use GitHub. Not shocking. :)

> let's say we block anonymous issue reporting

I think we get some good anonymous bug reports that we might not get otherwise. I also think we shouldn't feel bad about closing anon bugs that lack information. Given that we're not drowning in bugs (yet...), I'm not so worried about letting anon bugs in. And perhaps we can think of ways to float bugs with identity to the top and down-prioritize anon bugs. Or something.

> Let's say, the person has the choice to authenticate using Twitter to report bugs.

This could be cool. It would require some hacks though. But hacks are how we do anon reporting now. :)

**Comment by karlcow** [comment]
> The way I interpret this comment is that anonymous Webcompat bugs are not effective to let developers at The Verge know about bugs. 

I interpreted slightly differently, which is we have a process in place (with a support account). We usually answer only to bug reports through this process once the support team has done the triage and has all parameters. Then it is moved to the dev team.  I understand this process because in big companies you don't want to overload your dev team with a flood of contacts. 

> I agree with that -- it took @hallvors doing the outreach.

nope. @hallvors decided to do the outreach instead of me or @adamopenweb 
It's the way we are setup for all bugs. Nothing different here. I guess @hallvors knew already about the contact at Verge. It's unrelated to this issue.

Also agreed with the fact that most Verge readers don't have github accounts. ^_^ It's pretty clear. But GitHub had been chosen for the other side of the mirror such as specifically reaching the devs, not the reporters. The reporters on the cake is the icing on top of the cake.

hmmm I have the feeling that @miketaylr is not getting my prose. Maybe English? I started with `Maybe let's say we block ` aka brainstorming.

> This could be cool. It would require some hacks though. 

yes… I didn't say the opposite 😆 I wonder what's not working in our sentences, that we seem to say the same things but in ways which seem to contradict each other. 👹

**Comment by hallvors** [comment]
>>    I agree with that -- it took @hallvors doing the outreach.
> nope. @hallvors decided to do the outreach instead of me or @adamopenweb

s/@hallvors/somebody in Mike's initial statement and you do agree with each other :)

**Comment by miketaylr** [comment]
> yes… I didn't say the opposite 😆 I wonder what's not working in our sentences, that we seem to say the same things but in ways which seem to contradict each other. 👹

Nah, I'm just thinking out loud as I type. 

(obligatory I didn't say you said the opposite. heh)

So is anyone in favor of anonymous comments? Or do we want to change the title (or open a new bug) for figuring out twitter-auth for opening bugs/commenting?

**Comment by karlcow** [comment]
* Anonymous comments. Nope. 
* Pseudonymous maybe. Be twitter. Be a system where someone reporting a bug gets a unique key that they can use in followup comment for this specific bug or an email `to: bot+unique_id@webcompat.com` when they reply to this email, the system is able to treat it as an additional comments from the reporter on this bug or a link to a unique form page giving the possibility to further comment. There are many ways of proxyfying it.

**Comment by magsout** [comment]
yeah yeah OK, anonyous is bad because of spam, you're right. And yes twitter auth should be a good idea ..

>Pseudonymous maybe. Be twitter. Be a system where someone reporting a bug gets a unique key that they can use in followup comment for this specific bug or an email to: bot+unique_id@webcompat.com when they reply to this email, the system is able to treat it as an additional comments from the reporter on this bug or a link to a unique form page giving the possibility to further comment. There are many ways of proxyfying it. 

Love this way.. I don't how, but seems good ;)

**Comment by miketaylr** [comment]
Related to #457. Let's figure this out when we get there.
